api/app/src/main/kotlin/packit/service/OutpackServerClient.kt

```diff
@@ -0,0 +1,77 @@
+package packit.service
+
+import org.slf4j.LoggerFactory
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpMethod
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import packit.AppConfig
+import packit.model.OutpackMetadata
+import packit.model.OutpackResponse
+
+@Service
+class OutpackServerClient(appConfig: AppConfig)
+{
+
+    private val baseUrl: String = appConfig.outpackServerUrl
+
+    private val restTemplate = RestTemplate()
+
+    fun getChecksum(): String
+    {
+        return get("checksum")
+    }
+
+    fun getMetadata(): List<OutpackMetadata>
+    {
+        val url = "$baseUrl/metadata"
+        val response = restTemplate.exchange(
```

Comment:
Couldn't this use the `get` method? Also, not sure why `getMetadata` is in this PR - doesn't seem to be related to the scheduled job?

Reply:
You're right, it's used in the next one. Can remove from here. See comment on the `get` method - unfortunately passing around generic types doesn't work: https://github.com/mrc-ide/packit/pull/12/files/9448f2474fd6bface58247c8655124aab039696b#diff-07850fffb48c54cb4734c8aa8681a522c8fe3cd48c45d1620b42c4648e555040R41